services/ui-src/src/utils/other/mask.test.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  convertToCommaSeparatedString,
+  isCustomMask,
+  isNumberStringMaskable,
+  maskValue,
+} from "utils";
+
+const commaSeparatedMaskAcceptableTestCases = [
```

Comment:
All of these test cases are great, but just calling out again that comma-separated should be distinct from currency/decimal addition. There will be lots of cases where we don't want to have decimals added, for instance: "How many people..."